.eslintrc.js

```diff
@@ -2,15 +2,15 @@ module.exports = {
   root: true,
   env: {
     browser: true,
-    node: true
+    node: true,
   },
   extends: [
     'airbnb-base',
     '@nuxtjs/eslint-config-typescript',
     'plugin:nuxt/recommended',
-    'prettier'
+    'prettier',
   ],
   rules: {
-    "import/extensions": "off",
+    'import/extensions': 'off',
```

Comment:
我調試過，這樣 config 應該較合適
```suggestion
    'array-bracket-newline': [
      'warn',
      { minItems: 2 },
    ],
    'array-element-newline': [
      'warn',
      "always",
    ],
    'comma-dangle': [
      'warn',
      'always-multiline',
    ],
    'import/extensions': 'off',
```